content/en/docs/tasks/security/authorization/authz-custom/index.md

```diff
@@ -143,6 +143,7 @@ allows requests with the header `x-ext-authz: allow`.
             port: "4180" # The default port used by oauth2-proxy.
             includeRequestHeadersInCheck: ["authorization", "cookie"] # headers sent to the oauth2-proxy in the check request.
             headersToUpstreamOnAllow: ["authorization", "path", "x-auth-request-user", "x-auth-request-email", "x-auth-request-access-token"] # headers sent to backend application when request is allowed.
+            headersToDownstreamOnAllow:["content-type", "set-cookie"] # headers sent back to the client when request is allowed.
```

Comment:
Why this line needs to be added?

Reply:
This is to inform users that they can also set this option if needed, which already has been implemented.